src/SprykerEco/Zed/Econda/EcondaConfig.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Exception;
+use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\Econda\EcondaConstants;
+
+class EcondaConfig extends AbstractBundleConfig
+{
+    const ECONDA_CSV_DELIMITER = '|';
+    const ECONDA_CSV_CATEGORY_DELIMITER = '^^';
```

Comment:
These delimiters are hard-coded and not configurable?

Reply:
Yes, because those delimiters are not env or store specific.